libbeat/docs/processors-list.asciidoc

```diff
@@ -98,6 +98,9 @@ endif::[]
 ifndef::no_truncate_fields_processor[]
 * <<truncate-fields, `truncate_fields`>>
 endif::[]
+ifdef::no_translate_sid_processor[]
```

Comment:
I think you meant `ifndef` here.

Reply:
That's fixed in https://github.com/elastic/beats/pull/16961.